### PR TITLE
IGVF-1448-environmental-treatment

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/treatment.json [hunk 1]
```diff
@@ -299,7 +360,8 @@
             "type": "string",
```
Comment (gabdank) [inline src/igvfd/schemas/treatment.json:360]: Add comment in the treatment type, providing examples of chemical, protein and environmental treeatments.

File: src/igvfd/schemas/treatment.json [hunk 2]
```diff
@@ -128,34 +127,95 @@
...
+                            "required": [
+                                "treatment_term_id"
+                            ]
                         }
                     }
                 }
             ]
+        },
+        "treatment_term_name": {
+            "comment": "Environmental treatments are restricted to certain term names including: stiffness.",
```
Comment (gabdank) [inline src/igvfd/schemas/treatment.json:199]: Maybe "restricted to term names that include currently only stiffness."

**Review by gabdank — CHANGES_REQUESTED** [review]
add changelog update (minor changes) and other comments on PR

File: src/igvfd/schemas/treatment.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/treatment.json:360]: There already exist examples for chemical & protein, so I just added one for environmental.

File: src/igvfd/schemas/treatment.json (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/treatment.json:199]: done.

**Review by gabdank — APPROVED** [review]

**Review by gabdank — CHANGES_REQUESTED** [review]
update changelog

**Review by gabdank — APPROVED** [review]

**Pull request merged**